### PR TITLE
Build fix on Bacon CI

### PR DESCRIPTION
**Pull request by aarongranick-okta**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [guidelines](/okta/okta-oidc-js/blob/master/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Adding Tests
- [ ] Build related changes
- [x] CI related changes
- [ ] Documentation changes
- [ ] Other... Please describe:

## What is the current behavior?
Fails to detect CI environment on Bacon because `CI` variable is not set

## What is the new behavior?
Detects CI environment on Bacon using a different environment var

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: scripts/setup.sh [hunk 1]
```diff
@@ -28,13 +28,35 @@ OKTA_REGISTRY=${ARTIFACTORY_URL}/api/npm/npm-okta-master
...
+sed -i "s#${YARN_REGISTRY}#${OKTA_REGISTRY}#" packages/okta-vue/yarn.lock
+sed -i "s#${YARN_REGISTRY}#${OKTA_REGISTRY}#" packages/okta-react-native/yarn.lock
 
 if ! yarn install ; then
   echo "yarn install failed! Exiting..."
   exit ${FAILED_SETUP}
 fi
 
-# Revert the origional change
+# Revert the origional change(s)
```
Comment (vijetmahabaleshwar-okta) [inline scripts/setup.sh]: nit: fix already existing typo - origional -> original

File: scripts/setup.sh [hunk 2]
```diff
@@ -28,13 +28,35 @@ OKTA_REGISTRY=${ARTIFACTORY_URL}/api/npm/npm-okta-master
 #  - https://github.com/yarnpkg/yarn/issues/5892
 #  - https://github.com/yarnpkg/yarn/issues/3330
 
-# Replace yarn artifactory with Okta's
+# Replace yarn registry with Okta's
+echo "Replacing $YARN_REGISTRY with $OKTA_REGISTRY within yarn.lock files..."
 sed -i "s#${YARN_REGISTRY}#${OKTA_REGISTRY}#" yarn.lock
+sed -i "s#${YARN_REGISTRY}#${OKTA_REGISTRY}#" packages/okta-angular/test/e2e/harness/yarn.lock
```
Comment (vijetmahabaleshwar-okta) [inline scripts/setup.sh]: Do we need to update the lock files in test directory too?

**Comment by vijetmahabaleshwar-okta** [comment]
Right now, we're only replacing yarn registry links. But sometimes, people have used npm registry in  lock files, which causes failure on bacon. Not sure if we need it now, but something to keep in mind for the future.

File: scripts/setup.sh (hunk 1 above)
Comment (swiftone) [inline scripts/setup.sh]: nit: "original"?

File: scripts/setup.sh (hunk 2 above)
Comment (aarongranick-okta) [reply scripts/setup.sh]: yes

**Comment by aarongranick-okta** [comment]
> Right now, we're only replacing yarn registry links. But sometimes, people have used npm registry in lock files, which causes failure on bacon. Not sure if we need it now, but something to keep in mind for the future.
We are standardized on yarn within this repo. The samples sometimes use npm

**Pull request merged**